app/components/candidate_interface/course_choices_review_component.html.erb

```diff
@@ -1,6 +1,6 @@
 <% @application_choices.each do |application_choice| %>
   <div id='course-choice-<%= application_choice.id %>' class='qa-application-choice-<%= application_choice.id %> <%= warning_container_css_class(application_choice) %>'>
-  <% if @editable %>
+  <% if CandidateInterface::EndOfCyclePolicy.can_add_course_choice?(application_form: application_choice.application_form) && @editable %>
```

Comment:
I'd be tempted to remove the named parameter from this method signature, as it's pretty clear what we're passing in whenever we call it.